velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1793,7 +1793,10 @@ struct ToISO8601Function {
   ToISO8601Function() {
     auto formatter =
         functions::buildJodaDateTimeFormatter("yyyy-MM-dd'T'HH:mm:ss.SSSZZ");
-    VELOX_CHECK(!formatter.hasError(), "Default format should always be valid");
+    VELOX_CHECK(
+        !formatter.hasError(),
+        "Default format should always be valid, error: " +
```

Comment:
VELOX_CHECK takes fmt::format-style arguments

```
VELOX_CHEK(!formatter.hasError(), "Default ...: {}",  formatter.error().message());
```

This avoids generating error message strings in happy paths.

Reply:
@mbasmanova Thanks for your guidance. Updated.